pkgs/development/ocaml-modules/mlgmpidl/default.nix

```diff
@@ -10,10 +10,9 @@ stdenv.mkDerivation rec {
     sha256 = "17xqiclaqs4hmnb92p9z6z9a1xfr31vcn8nlnj8ykk57by31vfza";
   };
 
-  nativeBuildInputs = [ perl ocaml findlib mpfr camlidl ];
-  buildInputs = [ gmp mpfr ];
+  buildInputs = [ perl gmp mpfr ocaml findlib camlidl ];
 
-  strictDeps = true;
+  strictDeps = false;
```

Comment:
Also why?

Reply:
This might not be needed, let me test it again

Reply:
I figured out how to fix this properly